### DISCUSSION TITLE
Installing in HA

### DISCUSSION THREAD

**Discussion by rowanbradley** [emporia-vue-local/esphome#257] [Q&A]
I have tried to add the repository https://github.com/magico13/ha-emporia-vue to Home Assistant. It tells me it "is not a valid add-on repository". What am I doing wrong? How do I get this add-on to install?

Thank you - Rowan

**Comment by blalor** [comment]
That's a different project. It's also not an add-on, it's a custom component that can be installed with HACS. See further down that project's README: https://github.com/magico13/ha-emporia-vue#installation-with-hacs

**Comment by zanix** [comment]
If you flash this firmware to the EmporiaVue, you will use the ESPHome integration to add it to Home Assistant.
